tools/eslint-rules/required-modules.js

```diff
@@ -77,13 +77,13 @@ module.exports = function(context) {
           function(module) {
             return foundModules.indexOf(module === -1);
```

Comment:
Unrelated to this PR: Should this line be `return foundModules.indexOf(module) === -1;` instead?